docs/.vitepress/theme/custom.css

```
@@ -0,0 +1,392 @@
+/* ============================================================
+   Nightsky theme for Soar documentation.
+   A dark, twilight-inspired skin over the VitePress default
+   theme: deep charcoal background, cyan/blue accents, amber
+   inline code.
+   ============================================================ */
+
+/* --- Fonts --- */
+:root {
+  --vp-font-family-base: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI',
```

Stylelint (17.13.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)